x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -7,7 +7,7 @@
 import signalsMapping from './signals_mapping.json';
 import ecsMapping from './ecs_mapping.json';
 
-export const SIGNALS_TEMPLATE_VERSION = 3;
+export const SIGNALS_TEMPLATE_VERSION = 13;
```

Comment:
Bumped up with a gap of 10 for the `7.11` release following conversation with @elastic/security-detections-response to account for possible patch versions that are needed